MVPTask/Sources/NetworkService/NetworkService.swift

```diff
@@ -0,0 +1,80 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Интерфейс сервиса для получения данных из сети
+protocol NetworkServiceProtocol {
+    /// Получение рецептов
+    func getRecipes(
+        categoryName: String,
+        qParameter: String,
+        completion: @escaping (
+            Result<[Recipe]?, Error>
+        ) -> Void
+    )
+}
+
+/// Сервис для получения данных из сети
+final class NetworkService: NetworkServiceProtocol {
+    // MARK: Private Properties
+
+    private let jsonDecoder = JSONDecoder()
+
+    private var component = URLComponents()
+    private var scheme = "https"
+    private var host = "api.edamam.com"
+    private var path = "/api/recipes/v2"
```

Comment:
```suggestion
    private let scheme = "https"
    private let host = "api.edamam.com"
    private let path = "/api/recipes/v2"
```
  а вообще это хардкод
  как и ниже